code/game/objects/items/toys/toys.dm

```diff
@@ -1219,6 +1219,42 @@
 	icon_state = "fumoplushie"
 	pokephrase = "I just don't think about losing."
 
+//yall out here arguing squids v foxes when blahaj exists???
+/obj/item/toy/plushie/shark
+	name = "shark plushie"
+	desc = "Three feet of condensed love and recycled polyester. It's incredibly huggable."
+	w_class = ITEMSIZE_SMALL //it might be compressible but it's still 3ft long
+	icon_state = "blahaj"
```

Comment:
what

Reply:
shark plushie!

Reply:
"blahaj"?

Reply:
https://en.wikipedia.org/wiki/Bl%C3%A5haj

the internal icon_state name is 'blahaj' because blahaj is a type of large shark plushie made by IKEA which has gained a weirdly large following in the transgender community
due to this being many years in the future however, the word 'blahaj' never appears in anything IC 😔